Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm

```diff
@@ -1447,6 +1449,37 @@ - (void)_fetchDataOfTypes:(NSSet<NSString *> *)dataTypes completionHandler:(WK_S
     completionHandler(nil);
 }
 
+- (void)_restoreData:(NSData *)data completionHandler:(void(^)(void))completionHandler
```

Comment:
This method works fine today with one type.
But in the future - if two optional types exist - it will have to be robust against any of the types missing (e.g. not early return as error handling)

Not critical to account for with code now, but maybe a comment documenting for the future.

Reply:
That's a good point. I've updated the code to be better built for adding more types later as Chris recommended.